app/code/Magento/Sales/Block/Adminhtml/Order/Create/Newsletter.php

```diff
@@ -12,12 +12,10 @@
  * @author      Magento Core Team <core@magentocommerce.com>
  * @since 100.0.2
  */
-class Newsletter extends \Magento\Sales\Block\Adminhtml\Order\Create\AbstractCreate
+class Newsletter extends AbstractCreate
 {
     /**
-     * Constructor
-     *
-     * @return void
+     * @inheritdoc
```

Comment:
@engcom-Hotel, if I got right, you're suggesting reverting this change to whatever we had like this:
```suggestion
     * Constructor
     *
     * @return void
```

As to me it's not look logical, see reasoning below.

This method has 2 parent methods:
- `\Magento\Framework\View\Element\Template::_construct` which has absolutely the same return type, and also has better method description (because method `_construct` is not an actual constructor)
![image](https://github.com/magento/magento2/assets/1873745/3fbd05ca-8c9c-43aa-a06b-1316af40dfdd)
- `\Magento\Framework\View\Element\AbstractBlock::_construct` also have the same return type and good method description
![image](https://github.com/magento/magento2/assets/1873745/afe93b45-2e71-43f1-a5fc-649969431c5f)

So instead of reverting this change, I would accept it + report a bug to SVC check & fix it separately

Reply:
Hello @ihor-sviziev,

Thanks for the detailed explanation. I agree with your point. Let me send this PR for approval.

Thanks